pkgs/by-name/li/librespeed-go/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+let
+  version = "1.1.5";
+  src = fetchFromGitHub {
+    owner = "librespeed";
+    repo = "speedtest-go";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-ywGrodl/mj/WB25F0TKVvaV0PV4lgc+KEj0x/ix9HT8=";
+  };
+in
+buildGoModule {
+  pname = "librespeed-go";
+  inherit version src;
+
+  vendorHash = "sha256-ev5TEv8u+tx7xIvNaK8b5iq2XXF6I37Fnrr8mb+N2WM=";
+
+  ldflags = [
+    "-w"
+    "-s"
+  ];
+
+  postInstall = ''
+    cp -r web/assets $out/
+  '';
+
+  meta = {
+    description = "A very lightweight speed test implementation in Go.";
```

Comment:
This is a silly nixpkgs thing but descriptions aren't supposed to end in `.`  https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#meta-attributes